modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractJavaCodegen.java

```diff
@@ -668,10 +668,38 @@ public void processOpts() {
         additionalProperties.put(CONTAINER_DEFAULT_TO_NULL, containerDefaultToNull);
     }
 
+    private void postProcessMappingDiscriminators(Map<String, ModelsMap> objs) {
```

Comment:
can you please add a docstring explaining what this function does?

add a bit more code comment below explaining what the for loops do would be helpful.